cmd/admin-inspect.go

```diff
@@ -62,7 +62,12 @@ USAGE:
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
+EXAMPLES:
+  1. Download 'xl.meta' for a specific object from all the drives in a zip file.
+     {{.Prompt}} {{.HelpName}} myminio/bucket/test*/xl.meta
 
+  2. Download all constituent parts for a specific object, and optionally encrypt the downloaded zip.
```

Comment:
I believe a third example is needed with `**` to recursively download a folder from all disks

`mc admin inspect myminio/.minio.sys/**`